### ISSUE TITLE
Make the mobile app persistent

### ISSUE THREAD

**Issue by kerim** [Roam-Research/issues#78]
**Describe the problem**

I often have to re-sign in to my Roam when loading it on my mobile browser (iOS). I would like to be able to stay logged in on mobile, and load the last page I was looking at whenever I load the app, rather than getting bumped to the login screen.

**Feature implementation**

I go to the last page I was using and it is there.

**Comment by mbrochh** [comment]
Hmm this is strange. I have saved Roam as an app-icon in my home-view and before that, I had it open in Safari in tab for weeks and I can say that every time I click the icon (or I looked at the tab), it would show the loading icon and then show my database. I never had to log-in again. I do see the loading icon each time, but I never have to enter my password.

Did you install some plugins that wipe your cookies for privacy reasons? Or are you using Safari in incognito mode?

**Comment by kerim** [comment]
Correct, I don't have to enter my password, but I do have to click login. 
Also, because I have two databases, I have to choose from between them. 
So: (1) launch, (2) click logon, (3) choose database, (4) dismiss quick capture, (5) read text
What I'd prefer: (1) launch, (2) read text.
